apps/wallet/src/features/Swap/Success.tsx

```diff
@@ -0,0 +1,49 @@
+import type { ViewStyle } from 'react-native';
+import { StyleSheet } from 'react-native';
+import { Text, View } from '@walless/gui';
+import { Switch } from '@walless/icons';
+import { useSafeAreaInsets, useSnapshot } from 'utils/hooks';
+
+import { swapContext } from './context';
+
+const Success = () => {
+	const insets = useSafeAreaInsets();
+	const containerStyle: ViewStyle = {
+		...styles.container,
+		paddingTop: Math.max(insets.top, 20),
+		paddingBottom: 20,
+	};
```

Comment:
this is in-efficient, use `containerStyle = [styles.container, { ...dyanmicProps }];` instead.